Packs/RemoteAccess/Integrations/RemoteAccessv2/RemoteAccessv2_description.md

```diff
@@ -0,0 +1,9 @@
+## RemoteAccess v2
+
+This integration enables Cortex XSOAR to access and run commands on a terminal in a remote location (via SSH).
+
+### Ciphers
+- When ciphers are given, the SSH connection will be established with the first cipher of the given ciphers that given host supports.
+
+### Key Algorithms
+- When key algorithms are given, the SSH connection will be established with the first key algorithm of the given key algorithms that given host supports.
```

Comment:
what is the default behavior if they are not given?

Reply:
It picks some Key/Cypher that exists. TBH not sure how it decides to pick one, can try and investigate

Reply:
Please check and document